app/models/katello/content_override.rb

```diff
@@ -0,0 +1,36 @@
+module Katello
+  class ContentOverride
+    attr_accessor :content_label, :name, :value
+    ENABLEMENT_VALUES = ["yes", "no", "0", "1", "default"].freeze
+
+    def initialize(content_label)
+      @content_label = content_label
+    end
+
+    def enabled=(value = nil)
+      @name = "enabled"
+      @value = value
+    end
+
+    def to_hash
+      ret = {"content_label" => @content_label, "name" => @name}
+      ret["value"] = @value if @value
+      ret
+    end
+
+    def self.create_enabled(content_params)
```

Comment:
right, i guess i'd argue it'd make more sense to keep this 'limitation' at the api level.  Since our api only supports 'enabled', it'd call:

ContentOverride.new(content_label, 'enabled', value)